### ISSUE TITLE
Fall (2019) Semester Planning

### ISSUE THREAD

**Issue by jrtechs** [RITlug/tasks#122]
Week | Theme | Day | Title | Brief | Workshop | Presenter
-- | -- | -- | -- | -- | -- | --
1 | Intro | Aug 30 | Intro to RITlug | -- | -- | @RITlug/eboard-current 
2 | General Linux | Sep 6 | Everything SSH! | -- | SSH CTF | @jrtechs 
3 | General Linux | Sep 13 | Packaging talk | -- | -- | Jason Carr
4 | General Linux | Sep 20 | Guest Speaker | -- | -- | @Conan-Kudo
5 | General Linux | Sep 27 | Shells | -- | Pimp your shell | @RITlug/eboard-current 
6 | Hacktober | Oct 4 | Intro to Git | -- | -- | @Tjzabel 
7 | Hacktober | Oct 11 | History of FOSS | -- | -- | @ritjoe
8 | Hacktober | Oct 18 | Hacktoberfest Hackathon | -- | Yes | @RITlug/eboard-current 
9 | Hacktober | Oct 25 | PI Fest | -- | -- | @hkeena98 @jrtechs 
10 | Deep Dives | Nov 1 | Gaming on linux | -- | -- | @hkeena98 
11 | Deep Dives | Nov 8 |  Bash Scripting | | -- |  @Tjzabel 
12 | Deep Dives | Nov 15 | Intro to Linux Kernel Tracing |  | No | @acritelli
13 | Deep Dives | Nov 22 | Gitlab in the Federal Government | -- | -- | @Serubin
14 |  | Nov 29 | (Thanksgiving, no RITlug) | -- | -- | --
15 | Party | Dec 6 | -- | -- | -- | --
16 | -- | Dec 13 | (Finals, no RITlug) | -- | -- | --

**Comment by hkeena98** [comment]
Find speaker for September 20

**Comment by hkeena98** [comment]
Idea Pool for Speakers:
1. Flory 
2. Dan Shneidy 
3. Soloman 
4. Djoe

**Comment by hkeena98** [comment]
November Talk:
Anthony Critelli - Kernel Tracking

**Comment by jrtechs** [comment]
# 9/27 Shells talk
- Tim Bash
- Jeffery  ZSH
-  Henry Fish

Work shop?

**Comment by jrtechs** [comment]
@Serubin Tentatively wants to give a talk on November 8th

**Comment by Serubin** [comment]
To add details to this:
## 11/8 GovLab: GitLab in the Federal Government

Every day, the federal government tackles some of the most difficult challenges that we face as a society. From defending the country itself to ensuring law enforcement’s ability to protect citizens.

Inevitably, at the core of all of these challenges, is code.

In this talk, Solomon will take a brief look into how the federal government tackles these challenges on a daily basis including how it’s evolving: how it contributes to and develops open-source, how it utilizes Innersource within and across individual agencies, and of course, how GitLab sits at the middle of all of this.

TL;DR: An overview of OS in government and how gitlab helps the Gov OS mission.

**Comment by jrtechs** [comment]
I'm going to send out for this week's meeting(10-9-19).

**Comment by hkeena98** [comment]
Anthony Critelli, possible talk option for deep dive in November

**Comment by hkeena98** [comment]
Possible Talk Topic: Bash Scripting

**Comment by hkeena98** [comment]
Henry wants to do a talk at some point: Gaming in Linux

**Comment by hkeena98** [comment]
Possible Talk Topic: Nathaniel - Secure Linux Practices

**Comment by Serubin** [comment]
Hey - update - I need to switch dates, I'm unable to do November 8th due to a work commitment. Apologies.

**Comment by Tjzabel** [comment]
@Serubin thanks for the heads up. I've updated the calendar accordingly.

**Comment by hkeena98** [comment]
11/8/2019: Tim to give talk on Bash

**Comment by Serubin** [comment]
Final update - I apologize, I'm not going to be able to give this talk this month. I apologize for the last minute switching.

**Comment by jrtechs** [comment]
Fall semester has finished and was largely successful, I am greatly appreciative to everyone who was able to give a talk this semester.
